### ISSUE TITLE
User Settings Cog is missing

### ISSUE THREAD

**Issue by SouthernTanuki** [TakosThings/Fluent-Discord#337]
**Describe the bug**
The user settings button is missing its cog icon leaving the button blank.

**Screenshots**

<img width="109" height="89" alt="Image" src="https://github.com/user-attachments/assets/9324f514-3d72-4371-91ab-61dc97b35831" />

**Comment by rlawton1243** [comment]
Looks like Discord removed the `aria-label="User Settings"` from the button, so the theme can't find it right now.

If you're using the static theme, here's the fix. Change:

<img width="1410" height="200" alt="Image" src="https://github.com/user-attachments/assets/6fe73138-9337-43e0-993e-507ada0e72a9" />

To:

<img width="1452" height="231" alt="Image" src="https://github.com/user-attachments/assets/0febcd06-b961-45ac-847c-78580386bfeb" />

Getting rid of just one `aria-label` check should be fine, since the other buttons can overwrite with their specific parameters.
